cypress/page-objects/Signup.js

```diff
@@ -0,0 +1,33 @@
+/// <reference types="cypress" />
+
+export function addEmail(email) {
+  cy.get(':nth-child(2) > .form-control').type(email);
+}
+export function addUsername(username) {
+  cy.get(':nth-child(1) > .form-control').type(username);
+}
+export function addPassword(password) {
+  cy.get(':nth-child(3) > .form-control').type(password);
+}
+export function clickSignUpButton() {
+  cy.get('.btn').click();
+}
+export function signUpPage() {
+  cy.get(':nth-child(3) > .nav-link').click();
+}
+export function getErrorMessage(position, message) {
```

Comment:
this is a get method but also does a validation. If it is a get, it should return a value(in general, the object you are searching)